roles/lnls-ans-role-phoebus/defaults/main.yml

```diff
@@ -1,8 +1,9 @@
 ---
 phoebus_version: "{{ pkg_version_phoebus | default('4.6.1') }}"
-phoebus_repository: nightly
-phoebus_base_url: "https://controlssoftware.sns.ornl.gov/css_phoebus"
-phoebus_archive: "{{ phoebus_base_url }}/{{ phoebus_repository }}/phoebus-{{ phoebus_version }}-linux.zip"
+# phoebus_repository: nightly
+# phoebus_base_url: "https://controlssoftware.sns.ornl.gov/css_phoebus"
```

Comment:
I think we should use the previous sensible defaults for this. If wanted we can set this variable elsewhere and ansible will override it automatically